mobify/pinny
# Mobify velocity

Updated pinny to use the new mobify-velocity fork.

Status: **Ready to merge**

Reviewers: @kpeatt @tedtate @donnielrt 
## Changes
- pointed to new bower component for mobify-velocity
- updated build number
## How to test-drive this PR
- checkout this branch
- bower install && bower prune
- `grunt test`
- `grunt serve`
- click around the examples and check that pinny works
